recipe/bld.bat

```diff
@@ -0,0 +1,27 @@
+mkdir -p build
+pushd build
+
+cmake -G "Ninja" ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -DCMAKE_BUILD_TYPE=Release ^
+      -DBUILD_PYTHON_BINDINGS=ON ^
+      -DBUILD_CLI_EXECUTABLES=ON ^
+      -DBUILD_SHARED_LIBRARIES=ON ^
+      -DBUILD_TESTS=OFF ^
+      -DMATLAB_BINDINGS=OFF ^
+      -DDOWNLOAD_STB_IMAGE=OFF ^
+      -DDOWNLOAD_ENSMALLEN=OFF ^
+      -DPYTHON_EXECUTABLE=%PYTHON% ^
+      -DUSE_OPENMP=ON ^
+      -DENSMALLEN_INCLUDE_DIR=%LIBRARY_PREFIX%/include ^
+      -DARMADILLO_INCLUDE_DIR=%LIBRARY_PREFIX%/include ^
+      -DARMADILLO_LIBRARY=%PREFIX%/Library/lib/libarmadillo.so ^
```

Comment:
Hey, @marcelotrevisani I don't why I am getting thi error 
```
ninja: error: 'D:/bld/mlpack_1579978912287/_h_env/Library/lib/libarmadillo.so', needed by 'generate_pyx_adaboost.exe', missing and no known rule to make it
ninja: error: 'D:/bld/mlpack_1579978912287/_h_env/Library/lib/libarmadillo.so', needed by 'generate_pyx_adaboost.exe', missing and no known rule to make it
Traceback (most recent call last):
```
and at the same time build is also showing this
```
-- Found Armadillo: D:/bld/mlpack_1579978912287/_h_env/Library/lib/libarmadillo.so (found suitable version "9.200.4", minimum required is "8.400.0") 
-- Armadillo libraries: D:/bld/mlpack_1579978912287/_h_env/Library/lib/libarmadillo.so

```
Can you please explain why it is happening everything is done here except this.

Reply:
@himanshupathak21061998 I can see in `src/mlpack/bindings/python/CMakeLists.txt` that the build rule will link against all libraries in `${MLPACK_LIBRARIES}`:

```
  target_link_libraries(generate_pyx_${name} mlpack ${MLPACK_LIBRARIES})
```

Can you check to see what `MLPACK_LIBRARIES` is?  (You might need to write a patch for `CMakeLists.txt`.)  Alternately, you could try the variable `ARMADILLO_LIBRARIES`, not `ARMADILLO_LIBRARY`.  I am not sure if that would work.